scopesim/utils.py

```diff
@@ -726,6 +726,16 @@ def set_console_log_level(level="INFO"):
     update_logging()
 
 
+def set_inst_pkgs_path(pkg_path: Path | str) -> None:
+    """Set the local path for !SIM.file.local_packages_path (shortcut)."""
+    rc.__config__["!SIM.file.local_packages_path"] = str(pkg_path)
+
+
+def link_irdb(irdb_path: Path | str = Path("../irdb")) -> None:
```

Comment:
The default seems to be relative to the current working directory, which is not "by default that you cloned both repos in the same parent directory".

You'd need something like
```suggestion
def link_irdb(irdb_path: Path | str = Path(__file__).parent.parent.parent / "irdb") -> None:
```

Reply:
Well spotted. I ended up using the `rc.__pkg_dir__` as a basis now, which we had anyway and is shorter and also more robust in case we ever move this function somewhere deeper.

I was originally thinking about the line I most often end up using for this, which is either from an IPython console within the "ScopeSim" dir or a temp one adjacent to it...